rcfiles/jointLklDM.rc

```diff
@@ -33,7 +33,7 @@ jointLklDM.exportSvLogStep: TRUE
 jointLklDM.exportDataPath: exported_files
 
 # Path (will be added to all specified files and directories in this rcfile)
-jointLklDM.path: ./
+jointLklDM.path: .
```

Comment:
If you want, but it's not necessary as the macro/exec takes care properly of that and as of now it makes some ugly "//" in the output

Reply:
I did not know about that feature of jointLklDM, but still, I think it is better to add the final "/" so that it is clear at first sight that we are referring to a directory. I do not care that much about double or triple "/"